0006-zigzag-conversion.py

```diff
@@ -0,0 +1,17 @@
+from math import ceil
+class Solution:
+    def convert(self, s: str, numRows: int) -> str:
+        if numRows == 1:
+            return s
+        
+        n = len(s)
+        output = []
+        for r in range(numRows):
```

Comment:
same here spacing for every "for loop" you write make it appear and clearly visible.